gatsby_fp/netlify.toml

```diff
@@ -1,2 +1,2 @@
 [context.deploy-preview.environment]
-    PEOPLEAPI_HOST = 'api.stevens.edu/people/faculty'
+    PEOPLEAPI_HOST = 'api.stevens.edu/people-dev2/faculty'
```

Comment:
Why would we use people-dev2 here?

Reply:
This was made temporarily while we were testing on people-dev2. I was planning on changing this back upon approval of EK-428. @dmotan

Reply:
I see, I believe this should be fixed as am not sure if we want to continue with EK-428.